pages_domains.go

```diff
@@ -109,9 +110,10 @@ func (s *PagesDomainsService) GetPagesDomain(pid interface{}, domain string, opt
 // GitLab API docs:
 // // https://docs.gitlab.com/ce/api/pages_domains.html#create-new-pages-domain
```

Comment:
You didn't add it, but I just noticed the double `//` pair. Could you maybe remove the additional one while updating your PR? Thanks!